eRecruiter.Api/Responses/Mandator/PortalSettingsResponse.cs

```diff
@@ -19,12 +19,23 @@ public class PortalSettingsResponse
 
         #endregion General settings
 
+        #region Language Packs
+
+        public string[] LanguagePacks { get; set; }
```

Comment:
I would prefer a less specific collection type like ```IEnumerable<string>``` and a comment for the property.